packages/block-editor/src/components/block-settings-menu/block-settings-dropdown.js

```diff
@@ -226,6 +226,8 @@ export function BlockSettingsDropdown( {
 								<__unstableBlockSettingsMenuFirstItem.Slot
 									fillProps={ { onClose } }
 								/>
+							</MenuGroup>
+							<MenuGroup>
```

Comment:
This means all items added using `__unstableBlockSettingsMenuFirstItem` will be in a MenuGroup.